PocketKit/Sources/Analytics/AppEvents/Search.swift

```diff
@@ -303,6 +303,24 @@ public extension Events.Search {
         )
     }
 
+    /**
+     Fired when a user scrolls down the search list and triggers another results page call
+     */
+    static func searchResultsPage(
```

Comment:
still working with Analytics to determine what the details should be

Reply:
updated analytics in the following commit
`fix(analytics): update search page analytics`